web/scripts/generateZonesConfig.ts

```diff
@@ -124,7 +125,7 @@ const mergeRatioParameters = () => {
   return ratioParameters;
 };
 
-const writeJSON = (fileName: any, object: any) => {
+const writeJSON = (fileName: string, object: unknown) => {
```

Comment:
We should try and define the object type here later on but I relize it's not the simplest of types 😅

Reply:
Yeeeeeeah, I started doing that, but it becomes a type of anything that can be JSON.stringify'ed, and when looking in the native NodeJS types and around on the interwebs it seems like this was most common practice. And given the actual impact of this, I did not consider it worth pursuing further - but I'm very open to changing my mind on this 😄

Reply:
Since we are only writing 4 kinds of JSON objects (at least right now) we could make 4 types and then a union type for this object.

These would be zone, world, exchanges and the aggregated exchanges thing.

We should already have types for the zone at least right?

Reply:
Hmm, I started doing this, but now that I think of it I'm not sure we should limit what can be written to a JSON file - it feels counter-intuitive to be typing what output the "excluded exchanges" should be in here, when this function really shouldn't care about this 🤔 Happy to revisit this later though!

Reply:
Makes sense, and in any case the type errors should have been caught before we write things to json.